RetailCoder.VBE/Refactorings/ReorderParameters/ReorderParametersRefactoring.cs

```diff
@@ -151,7 +151,7 @@ private void RewriteCall(VBAParser.ArgumentListContext argList, ICodeModule modu
             var args = argList.argument().Select((s, i) => new { Index = i, Text = s.GetText() }).ToList();
             for (var i = 0; i < _model.Parameters.Count; i++)
             {
-                if (argList.argument().Count <= i)
+                if (argList.argument().Count() <= i)
```

Comment:
If that's an array, better use its `Length` property. If it's not an array, ignore this comment :)